tests/e2e-tests/pom.xml

```diff
@@ -75,6 +101,50 @@
 
    <build>
       <plugins>
+         <plugin>
```

Comment:
Can you add a TODO here to replace codehaus for something better.. I think you opened a JIRA on this.

Reply:
I'm confused though.. I thought we needed to replace codehaus.. but that seems someone still maintaining it on GitHub? 


If you could add just a TODO now so we can figure out this later?

Reply:
Mojo bits from Codehaus were then established as/at Mojohaus, and yet kept the groupId.